extensions/wikia/ImageReviewMercury/ImageReviewMercury.i18n.php

```diff
@@ -0,0 +1,9 @@
+<?php 
+
+$messages = array();
+
```

Comment:
Please, add `$messages['qqq']` array and document messages. Message documentation is to give other people the context of the message to avoid misunderstandings. Let's take "Comment" as an example. It can be a noun used in a purely informative header. It can also be a verb used in imperative mood in a submit button. It would not make sense to translate the first one as "Prześlij komentarz", while it does make perfect sense on the latter.

It might not be a super important here, but consider it as a generally good practice in terms of overall software i18n and l10n.